spring-boot-starter-camunda/src/main/resources/application-camunda-oidc.yaml

```diff
@@ -10,6 +10,7 @@ camunda:
       enabled: true
       base-url: http://localhost:26500
       audience: zeebe-api
+      rest-address: http://localhost:8080
```

Comment:
This needs to be aligned with the docker-compose from camunda/camunda-platform

Reply:
indeed, it is still not defined which port will be used for Zeebe REST in camunda-platform